### PR TITLE
Community Contribution: Prevent exception on undefined trigger handler action

### LINKED ISSUE #2742: Trigger Handler Null Pointer Exception when the Trigger Action is empty
The trigger action field is not required and can be nulled out by an end user. This will result in an error message for both Health Check and when the trigger is ran on the object: "System.NullPointerException: Attempt to de-reference a null object Class.TDTM_ObjectDataGateway.getClassesToCallForObject: line 74, column 1 External entry point Class.TDTM_TriggerHandler.run: line 89, column 1 Trigger.TDTM_Account: line 34, column 1"

### PR DESCRIPTION
**Pull request by RadSFDeveloper**
# Critical Changes

# Changes
-  Display NPSP Settings page and run Health Check when a trigger handler having undefined action exists. Thank you Manik (@manikrai29) for your contribution!

# Issues Closed
- Fixes #2742 

# New Metadata
Apex Class:
- TDTM_ObjectDataGateway_TEST

# Deleted Metadata

### CODE REVIEW

**Comment by RadSFDeveloper** [comment]
@sam-knox , can you please review "Changes" section?

**Comment by RadSFDeveloper** [comment]
**lurch: attach W-020423

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x80000008F6wYAAS" target="_blank">W-020423</a>

File: src/classes/TDTM_ObjectDataGateway_TEST.cls-meta.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>43.0</apiVersion>
```
Comment (force2b) [inline src/classes/TDTM_ObjectDataGateway_TEST.cls-meta.xml]: This can be api v46.

Comment (RadSFDeveloper) [reply src/classes/TDTM_ObjectDataGateway_TEST.cls-meta.xml]: Changed version to 46. It was 43 initially since `TDTM_ObjectDataGateway` is v43.

**Comment by Julian88Tex** [comment]
[WI](https://foundation.lightning.force.com/lightning/r/agf__ADM_Work__c/a2x80000008F6wYAAS/view) merged to master.

**Pull request merged**